ddr/include/ddr/scanner/dwarf/DwarfFunctions.hpp

```diff
@@ -324,3 +327,4 @@ int dwarf_dieoffset(Dwarf_Die die, Dwarf_Off *dieOffset, Dwarf_Error *error);
 int dwarf_get_TAG_name(Dwarf_Half tag, const char **name);
 
 void setError(Dwarf_Error *error, Dwarf_Half num);
+#endif /*DWARFFUNCTIONS_HPP*/
```

Comment:
A blank line before this `#endif` would be good (and would balance the opening `#ifndef`).